### PR TITLE
release 0.3.0

### PR DESCRIPTION
**Pull request by domenicrosati**
## Reason for change

Release version flag and a CTA change

## Testing

- [x] Testing RC out in console-plan-builder

## Checklist

- [x] Release rc
- [x] [CHANGELOG](./CHANGELOG.md) updated

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
 
+## [0.3.0] - 2020-04-17
```
Comment (dangodev) [inline CHANGELOG.md:10]: 👍

File: CHANGELOG.md [hunk 2]
```diff
@@ -7,6 +7,8 @@ adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```
Comment (dangodev) [inline CHANGELOG.md:8]: Need to delete the “Unreleased” line for release, then add it back again for the next feature

Comment (domenicrosati) [reply CHANGELOG.md:8]: ooo i thought we kept it will do

**Comment by domenicrosati** [comment]
@dangodev so do i release it when the PR is a approved or mergerd or just do it now that i've tested it

**Comment by dangodev** [comment]
> so do i release it when the PR is a approved or mergerd or just do it now that i've tested it

The typical flow is to tag a `v0.3.0-rc.0` from this branch, before merging, and you have a place where you install and test it. That’s good to do because something can go wrong in packaging/publishing, and that’s really what you’re testing more than your code

Once that works, you can merge & tag `v0.3.0` from `master`

**Comment by domenicrosati** [comment]
Ok sweet thanks ya I am testing it locally:
Screen Shot 2020-04-17 at 9 56 42 AM

**Comment by dangodev** [comment]
The reason we want to be overly-diligent about testing is because you can’t roll back or unpublish from npm. And you can’t do that to our CDN either. So every major release basically streams live to providers’ websites and there’s a lot of risk and potential $ loss with each release

**Pull request merged**